src/python/CMakeLists.txt

```diff
@@ -38,12 +38,33 @@ execute_process (
 file (TO_CMAKE_PATH "${PYTHON_PREFIX}" PYTHON_PREFIX)
 execute_process (
    COMMAND ${PYTHON_EXECUTABLE} -c
-       "import site, sys; sys.stdout.write(site.getusersitepackages().replace(site.getuserbase(), site.PREFIXES[-1]))"
+       "	\n
+import site, sys \n
+dirs = site.getsitepackages(); \n
+dir = dirs[0] \n
+for d in dirs: \n
+	if d.count(\"packages\"): \n
+		dir = d[d.index(\"packages\")-5:] \n
```

Comment:
this "-5" will break if the directory for modules is called e.g. "usersite-packages", i.e. when the part before "packages" is not five characters long.